plugins/main/bridge/bridge.go

```diff
@@ -93,7 +94,8 @@ func init() {
 
 func loadNetConf(bytes []byte, envArgs string) (*NetConf, string, error) {
 	n := &NetConf{
-		BrName: defaultBrName,
+		BrName:              defaultBrName,
+		PreserveDefaultVlan: true,
 	}
```

Comment:
How about to add some comments to mention the defalt value, true, comes from backward compatibility, such as 'default is true for compatibility for previous version'